webui/src/pages/admin-dashboard/namespace-change-dialog.tsx

```diff
@@ -0,0 +1,121 @@
+         };
+     }, []);
+
+    useEffect(() => {
+        if (open) {
+            setNewNamespace('');
+            setKeepOldNamespace(false);
+            setAbortIfNewNamespaceAlreadyExists(false);
+        }
+    }, [open]);
+
+    const onClose = () => {
+        props.onClose();
+    };
+    const onKeepOldNamespaceChange = (event: React.ChangeEvent, checked: boolean) => {
+        setKeepOldNamespace(checked);
+    };
+    const onAbortIfNewNamespaceAlreadyExistsChange = (event: React.ChangeEvent, checked: boolean) => {
+        setAbortIfNewNamespaceAlreadyExists(checked);
+    };
+    const handleChangeNamespace = async () => {
+        try {
+            if (!props.namespace) {
+                return;
+            }
+            setWorking(true);
+            props.setLoadingState(true);
+            const oldNamespace = props.namespace.name;
+            const result = await service.admin.changeNamespace(abortController, { oldNamespace, newNamespace, keepOldNamespace, abortIfNewNamespaceAlreadyExists });
+            if (isError(result)) {
+                throw result;
+            }
+            props.setLoadingState(false);
+            setWorking(false);
+            props.onClose(newNamespace);
+        } catch (err) {
+            props.setLoadingState(false);
+            setWorking(false);
+            handleError(err);
+        }
+    };
+
+     return <>
+        <Dialog onClose={onClose} open={open} aria-labelledby='form-dialog-title'>
+            <DialogTitle id='form-dialog-title'>Change Namespace</DialogTitle>
+            <DialogContent>
+                <DialogContentText>
+                     Enter the new Namespace name.
+                </DialogContentText>
+                <TextField
+                    autoFocus
+                    margin='dense'
+                    id='name'
+                    autoComplete='off'
+                    label='New Open VSX Namespace'
+                    fullWidth
+                    onChange={(event) => {
+                        setNewNamespace(event.target.value);
+                    }}
+                />
+                <FormControlLabel
+                    control={<Checkbox checked={keepOldNamespace} onChange={onKeepOldNamespaceChange} name='keep-old-namespace' />}
+                    label={`Keep old '${props.namespace.name}' namespace`} />
+                <FormControlLabel
+                    control={<Checkbox checked={abortIfNewNamespaceAlreadyExists} onChange={onAbortIfNewNamespaceAlreadyExistsChange} name='abort-change-namespace' />}
+                    label='Abort the process if new namespace already exists' />
+             </DialogContent>
+             <DialogActions>
+                 <Button
+                    variant='contained'
+                    color='primary'
+                    onClick={onClose} >
+                    Cancel
+                </Button>
+                <ButtonWithProgress
+                    autoFocus
```

Comment:
nit (ux): We already have `autoFocus` on line 88, and I think there it makes more sense (you need to input something before clicking this). 
```suggestion
```